### PR TITLE
Command Line Interface created.

### PR DESCRIPTION
**Pull request by anderson89marques**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,5 @@ clonagem e execução
 - Você deve primeiro criar um ambiente virtual usando o python 3.5 (python -m venv .ven, se você não usa pyenv será python3.5 -m venv .venv) 
 - Ativar o ambiente virtual(source .venv/bin/activate)
 - Executar pip install -r requirements.txt
-- Executar python main.py
\ No newline at end of file
+- Executar o comando do flit para criar um executável: flit install --symlink --python python
+- Executar, ex: binance -i 1m
```
Comment (samuelhei) [inline README.md:11]: Why is the name of the executable binance? I think this name has a great chance to conflict with other packages...

But we can discuss this in another opportunity.

File: binance/cli.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        description="Python tool to download Binance Candlestick (k-line) data from REST API"
+    )
+    parser.add_argument('--interval', '-i',
+                        help="""frequence interval in minutes(m); hours(h); days(d); weeks(w); months(M);
+             all possibles values: 
+             1m 3m 5m 15m 30m 1h 2h 4h 6h 8h 12h 1d 3d 1w 1M""", required=True)
+    parser.add_argument(
+        '--symbol', '-s', help="pair. default: 'ETHBTC'.", default='ETHBTC')
+    parser.add_argument(
+        '--limit', '-l', help="quantity of items returned; max 500; default 500.")
```
Comment (samuelhei) [inline binance/cli.py:21]: The limit of 500 rows is per request on binance API, we need to get around this creating multiple requests and biding then on the same output file.

**Review by samuelhei — CHANGES_REQUESTED** [review]
Nice improvements! The only issue that we need to attend before approve this PR is the limitations of the 500 rows per query (explained better in comments).

File: README.md (hunk 1 above)
Comment (anderson89marques) [reply README.md:11]: ok, we discussed it later.

File: binance/cli.py (hunk 2 above)
Comment (anderson89marques) [reply binance/cli.py:21]: I'll create another task to deal with this issue.

**Pull request merged**